Assets/Scripts/TXT/MemoriesParser.cs

```diff
@@ -0,0 +1,37 @@
+using UnityEngine;
+using UnityEngine.UI;
+using System.Collections.Generic;
+using System;
+using System.IO;
+/* Author: Anisha Pai
+* Description: Retrieves a memory with method getMemory(int id)
+*/
+
+public class MemoriesParser: MonoBehaviour {
+
+    public TextAsset jsonMemory;
+    MemoryGroup allMemories;
+
+    public void Awake() {
+
+        jsonMemory = (TextAsset)Resources.Load("jsonMemories") as TextAsset;
+        allMemories = JsonUtility.FromJson<MemoryGroup>(jsonMemory.text);
+    }
+
+    public String getMemory(int id) {
```

Comment:
@imann24 Made the changes! Just realised sourcetree is logged into your account, so maybe you didn't get notified of the pull request.